src/core/core.helpers.js

```diff
@@ -481,8 +481,15 @@ module.exports = function() {
 
 		return padding.indexOf('%') > -1 ? parentDimension / parseInt(padding, 10) : parseInt(padding, 10);
 	};
+	helpers.getParentNode = function(domNode) {
```

Comment:
This method should be private (and at some point will be moved in the dom platform implementation):

```javascript
/**
 * @private
 */
helpers._getParentNode = function(node) {
```